ci/format_pre.sh

```diff
@@ -63,5 +63,11 @@ if [[ "${#FAILED[@]}" -ne "0" ]]; then
     for failed in "${FAILED[@]}"; do
         echo "${BASH_ERR_PREFIX} $failed" >&2
     done
+    if [[ $(git status --porcelain) ]]; then
+        git diff > "$DIFF_OUTPUT"
+        echo
+        echo "Diff file with (some) fixes will be uploaded. Please check the artefacts for this PR run in the azure pipeline."
```

Comment:
Can you programmatically generate the URL here? As a personal pain point, hunting around AZP results for things like coverage results, docs, is a PITA. It would be nice if we could have some way (ideally a single click away from the PR on GH) to get at these artefacts.

Reply:
Sure, let's ship this, but it would be great to have this improvement.